src/java/org/pantsbuild/tools/junit/impl/ConsoleRunnerImpl.java

```diff
@@ -281,6 +281,7 @@ public void testStarted(Description description) throws Exception {
         default:
           throw new IllegalStateException();
       }
+      suiteCapture.incrementUseCount();
```

Comment:
I might be wrong, but I think that useCount isn't necessary anymore as the only thing that callls close() on the suite capture is dispose() which is called at the end of the run.